### PR TITLE
Use DJANGO_SETTINGS_MODULE to indicate which settings to use

### PR DESCRIPTION
**Pull request by rstorey**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -41,19 +36,17 @@ Instead of doing ``docker-compose up`` as above, instead do the following::
...
 
     $ python3 -m venv env
     $ source env/bin/activate
     $ pip install -r requirements_devel.txt
 
-Edit, or create the config/config-optional-override.json and set the file content to: {"mode":"mac"}
-
-
-Finally, run migrations and launch the development server::
+Finally, configure the Django settings, run migrations and launch the development server::
```
Comment (acdha) [inline README.rst]: Oxford comma

File: concordia/celery.py [hunk 2]
```diff
@@ -5,7 +5,7 @@
 from celery import Celery
 
 # set the default Django settings module for the 'celery' program.
-os.environ.setdefault("DJANGO_SETTINGS_MODULE", "concordia.settings")
+# os.environ.setdefault("DJANGO_SETTINGS_MODULE", "concordia.settings")
```
Comment (acdha) [inline concordia/celery.py]: Seems like we could just delete this line entirely?

File: concordia/settings_dev.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+ 
+CONCORDIA = dict(netloc="http://0.0.0.0:8000") 
+ 
+
+IMPORTER = {
+    "BASE_URL"       :   "",
+    "IMAGES_FOLDER"  :   "/concordia_images/",
+    "ITEM_COUNT"     :   "",
+    "S3_BUCKET_NAME" :   ""
+}
```
Comment (acdha) [inline concordia/settings_dev.py]: Trailing newline at the end of the file — Black should take care of that

File: concordia/settings_prod.py [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+# TODO: For final deployment to production, when we are running https, uncomment this next line
+# CSRF_COOKIE_SECURE = True
+
+IMPORTER = {
+    "BASE_URL"       :   "",
+    "IMAGES_FOLDER"  :   "/app/",
+    "ITEM_COUNT"     :   "",
+    "S3_BUCKET_NAME" :   ""
+}
```
Comment (acdha) [inline concordia/settings_prod.py]: Trailing newline

File: concordia/settings_prod.py [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+ 
+LOGGING["handlers"]["stream"]["level"] = "INFO"
+LOGGING["handlers"]["file"]["level"] = "INFO"
+LOGGING["handlers"]["file"]["filename"] = "./logs/concordia-web.log"
+LOGGING["handlers"]["celery"]["level"] = "INFO"
+LOGGING["handlers"]["celery"]["filename"] = "./logs/concordia-celery.log"
+LOGGING["loggers"]["django"]["level"] = "INFO"
+LOGGING["loggers"]["celery"]["level"] = "INFO"
+
+DJANGO_SECRET_KEY = "np*no_id-t8*if4*a(pz(n(rk5#(rlyn-41jvp5u%*ij&5u-3x"
```
Comment (acdha) [inline concordia/settings_prod.py]: I generally like setting these to something which makes it obvious that this is not the value which will actually be deployed

File: concordia/settings_template.py [hunk 6]
```diff
@@ -60,18 +48,18 @@
...
 
 DATABASES = {
     "default": {
-        "ENGINE": Config.Get("database")["adapter"],
-        "NAME": Config.Get("database")["name"],
-        "USER": Config.Get("database")["username"],
-        "PASSWORD": Config.Get("database")["password"],
-        "HOST": Config.Get("database")["host"],
-        "PORT": Config.Get("database")["port"],
+        "ENGINE": "django.db.backends.postgresql_psycopg2",
```
Comment (acdha) [inline concordia/settings_template.py]: Isn't `django.db.backends.postgresql` preferred these days? https://docs.djangoproject.com/en/2.0/ref/settings/#std:setting-DATABASE-ENGINE

File: concordia/settings_template.py [hunk 7]
```diff
@@ -216,11 +200,11 @@
 ACCOUNT_ACTIVATION_DAYS = 7
 
 REST_FRAMEWORK = {
-    "PAGE_SIZE": Config.Get("djrf")["PAGE_SIZE"],
+    "PAGE_SIZE": 10,
     "DEFAULT_PAGINATION_CLASS": "rest_framework.pagination.PageNumberPagination",
 }
 
-CONCORDIA = dict(netloc=Config.Get("concordia")["NETLOC"])
+CONCORDIA = dict(netloc="http://0.0.0.0:80")
```
Comment (acdha) [inline concordia/settings_template.py]: Generally I'd prefer `{"netloc": "http://0.0.0.0:80"}`

File: concordia/settings_test.py [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+LOGGING["handlers"]["celery"]["level"] = "DEBUG"
+LOGGING["handlers"]["celery"]["filename"] = "./logs/concordia-celery.log"
+LOGGING["loggers"]["django"]["level"] = "DEBUG"
+LOGGING["loggers"]["celery"]["level"] = "DEBUG"
+
+DEBUG = True
+
+DATABASES = {
+    "default": {
+        "ENGINE": "django.db.backends.postgresql_psycopg2",
```
Comment (acdha) [inline concordia/settings_test.py]: `django.db.backends.postgresql`

File: concordia/settings_test.py [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+    }
+}
+
+ALLOWED_HOSTS = ["127.0.0.1","0.0.0.0"]
+
+CONCORDIA = dict(netloc="http://0.0.0.0:8000")
+
+IMPORTER = {
+    "BASE_URL"       :   "",
+    "IMAGES_FOLDER"  :   "/concordia_images/",
```
Comment (acdha) [inline concordia/settings_test.py]: For testing, etc. should this default to something under `/tmp`?

File: concordia/wsgi.py [hunk 10]
```diff
@@ -11,6 +11,6 @@
 
 from django.core.wsgi import get_wsgi_application
 
-os.environ.setdefault("DJANGO_SETTINGS_MODULE", "concordia.settings")
+# os.environ.setdefault("DJANGO_SETTINGS_MODULE", "concordia.settings")
```
Comment (acdha) [inline concordia/wsgi.py]: Thinking that we should  delete this if we no longer need it

File: importer/importer/celery.py [hunk 11]
```diff
@@ -5,7 +5,7 @@
 from celery import Celery
 
 # set the default Django settings module for the 'celery' program.
-os.environ.setdefault("DJANGO_SETTINGS_MODULE", "concordia.settings")
+# os.environ.setdefault("DJANGO_SETTINGS_MODULE", "concordia.settings")
```
Comment (acdha) [inline importer/importer/celery.py]: Delete this if we don't need it

File: importer/importer/models.py [hunk 12]
```diff
@@ -9,21 +9,11 @@
...
-BASE_DIR = os.path.dirname(PROJECT_DIR)
-
-sys.path.append(BASE_DIR)
-
-sys.path.append(os.path.join(BASE_DIR, "config"))
-from config import Config
-
-
-# TODO: use util to import Config
+from django.conf import settings
```
Comment (acdha) [inline importer/importer/models.py]: I'd run the imports through `isort`

File: manage.py [hunk 13]
```diff
@@ -3,7 +3,7 @@
 import sys
 
 if __name__ == "__main__":
-    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "concordia.settings")
+    # os.environ.setdefault("DJANGO_SETTINGS_MODULE", "concordia.settings")
```
Comment (acdha) [inline manage.py]: Delete this if we don't need it

**Comment by acdha** [comment]
I don't know what editor you use but Black has been the first thing where I felt safe enabling VSCode's format-on-save for Python. It's liberating!

**Comment by rstorey** [comment]
I actually just switched to VSCode after you mentioned it to me last week. I'm not sure how to enable format-on-save but I'll look into it. I'll also recommend the rest of the team to do the same thing tomorrow morning.

**Pull request merged**